src/components/RequestsFilters/RequestsFiltersConfig.test.js

```diff
@@ -0,0 +1,47 @@
+import { requestFilterTypes } from '../../constants';
+import filtersConfig from './RequestsFiltersConfig';
+
+describe('RequestsFiltersConfig', () => {
+  it('should have a filter for requestType', () => {
+    const requestTypeFilter = filtersConfig.find(f => f.name === 'requestType');
+    expect(requestTypeFilter).toBeTruthy();
+    expect(requestTypeFilter.cql).toEqual('requestType');
+    expect(requestTypeFilter.values).toEqual([]);
+    expect(requestTypeFilter.operator).toEqual('==');
```

Comment:
Could you change those "expect" statements to only one "expect"?
```suggestion
    const expectedResult = {
    name: 'requestType',
    cql: 'requestType',
    values: [],
    operator: '==',
    };

    expect(requestTypeFilter).toEqual(expectedResult);
```

Could you do the same with other tests?

Reply:
All tests updated